pm/sendermonitor.go

```diff
@@ -82,11 +82,15 @@ type senderMonitor struct {
 	// each of currently active remote senders
 	redeemable chan *SignedTicket
 
+	// gasPriceUpdate is a channel used to receive notifications
+	// of gas price changes
```

Comment:
Could be one line comment as well but I'm really just nitpicking when it comes to that

Reply:
Fixed in 0130b20